adafruit_pcf8591/analog_out.py

```diff
@@ -26,16 +26,21 @@
 
 * Author(s): Bryan Siepert
 """
+try:
+    import typing  # pylint: disable=unused-import
+    from adafruit_pcf8591.pcf8591.PCF8591 import PCF8591
+except ImportError:
+    pass
 
 
 class AnalogOut:
     """AnalogIn Mock Implementation for ADC Reads."""
 
-    def __init__(self, pcf, dac_pin=0):
+    def __init__(self, pcf: PCF8591, dac_pin: int = 0) -> None:
         """AnalogIn
 
         :param pcf: The pcf object.
-        :param ~digitalio.DigitalInOut DAC pin: Required pin must be P4
+        :param adafruit_pcf8591.pcf8591.A0 pin: Required pin must be A0
```

Comment:
This should make the parameter show up in RTD as integer, and should link it to the correct constant as well, though Sphinx may throw a fit if it thinks the reference isn't specific enough.  We can add some more stuff to fix it in that case.

```suggestion
:param int pin: Required pin must be ~adafruit_pcf8591.pcf8591.A0
```

Reply:
the param pcf doesn't need expanded?

Reply:
It should be, but we can also do it in another PR as well.  At least for now it renders correctly.